backend/src/classes/SocketSession.ts

```diff
@@ -0,0 +1,16 @@
+import Duxcore from "../Duxcore";
+
+export class SocketSession {
+	public client: Duxcore;
+
+	private _user?: string;
+
+	constructor(client: Duxcore) {
+		this.client = client;
+	}
+
+	get permissions(): String[] {
+		return [];
+	}
+
```

Comment:
inconsistency - other classes have a \n\n at the start

Reply:
The hell are you on about?